pkgs/top-level/python-packages.nix

```diff
@@ -17148,11 +17217,12 @@ in modules // {
 
 
   pyparsing = buildPythonPackage rec {
-    name = "pyparsing-2.0.1";
+    name = "pyparsing-${version}";
+    version = "2.1.0";
 
     src = pkgs.fetchurl {
       url = "http://pypi.python.org/packages/source/p/pyparsing/${name}.tar.gz";
-      sha256 = "1r742rjbagf2i166k2w0r192adfw7l9lnsqz7wh4mflf00zws1q0";
+      sha256 = "0099k2aj7y7bd4kgkd2nj6ah6af0xkvlgnwrqv1lf4s9bb42pjzn";
     };
 
     # error: invalid command 'test'
```

Comment:
It is not quite the case anymore, but the tests causes various problems.

I’ll look into it ASAP.